test-common/src/main/scala/org/scalajs/testcommon/RPCCore.scala

```diff
@@ -262,10 +262,10 @@ private[scalajs] object RPCCore {
   type OpCode = Byte
 
   /** Exception thrown if a remote invocation fails. */
-  class RPCException(c: Throwable) extends Exception(null, c)
+  class RPCException(c: Throwable) extends Exception(c.getMessage, c)
```

Comment:
Consider simply using `Exception(c)`, using the constructor [`Exception(Throwable cause)`](https://docs.oracle.com/javase/8/docs/api/java/lang/Exception.html#Exception-java.lang.Throwable-) which will automatically incorporate the cause's `toString()` as the `RPCException`'s message.

Reply:
True, that's neater.